app/src/main/cpp/skyline/kernel/svc.cpp

```diff
@@ -584,8 +584,7 @@ namespace skyline::kernel::svc {
     }
 
     void ConnectToNamedPort(DeviceState &state) {
-        constexpr auto portSize = 0x8; //!< The size of a port name string
-        std::string_view port(state.process->GetPointer<char>(state.ctx->registers.x1), portSize);
+        std::string_view port(state.process->GetPointer<char>(state.ctx->registers.x1), sizeof(service::ServiceName));
```

Comment:
string view doesn't deal with null terminators afaik? we should either use the thing in span I suggested adding or maybe split it out to a helper